docs/script-console-scripts/configure-yadocker-cloud.groovy

```diff
@@ -0,0 +1,417 @@
+   deal in the Software without restriction, including without limitation the
+   rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+   sell copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+   IN THE SOFTWARE.
+ */
+/*
+   Configure Yet Another Docker clouds provided by the Yet Another Docker
+   Plugin.  This Jenkins Script Console script makes it easy to maintain large
+   amounts of configured clouds.
+
+   Note: This script deletes all yet another docker cloud configurations from
+   Jenkins before configuring the clouds.  It does not affect in-process
+   builds.  However, if there are previously configured yet another docker
+   clouds then they will be removed.
+
+   Yet Another Docker Plugin 0.1.0-rc31
+ */
+
+import com.github.kostyasha.yad.DockerCloud
+import com.github.kostyasha.yad.DockerConnector
+import com.github.kostyasha.yad.DockerContainerLifecycle
+import com.github.kostyasha.yad.DockerSlaveTemplate
+import com.github.kostyasha.yad.commons.DockerCreateContainer
+import com.github.kostyasha.yad.commons.DockerImagePullStrategy
+import com.github.kostyasha.yad.commons.DockerPullImage
+import com.github.kostyasha.yad.commons.DockerRemoveContainer
+import com.github.kostyasha.yad.commons.DockerStopContainer
+import com.github.kostyasha.yad.launcher.DockerComputerJNLPLauncher
+import com.github.kostyasha.yad.launcher.DockerComputerLauncher
+import com.github.kostyasha.yad.launcher.DockerComputerSSHLauncher
+import com.github.kostyasha.yad.other.ConnectorType
+import com.github.kostyasha.yad.strategy.DockerOnceRetentionStrategy
+
+import hudson.model.Node
+import hudson.plugins.sshslaves.SSHConnector
+import hudson.slaves.EnvironmentVariablesNodeProperty
+import hudson.slaves.NodeProperty
+import hudson.slaves.RetentionStrategy
+import hudson.tools.ToolLocationNodeProperty
+import jenkins.model.Jenkins
+import net.sf.json.JSONArray
+import net.sf.json.JSONObject
+
+/*
+   Configure the Yet Another Docker Plugin via this clouds_yadocker variable.
+
+   This variable can be removed and referenced for other configurations.  All
+   values are optional with defaults set.
+ */
+
+JSONArray clouds_yadocker = [
+    //YET ANOTHER DOCKER CLOUD (this is an item in a list of clouds)
+    [
+        cloud_name: "docker-local",
```

Comment:
why do you need JSON? Isn't easier to configure it like max from riot do? I tried to have fluent setters https://github.com/stapler/stapler/pull/81 but failed with stapler magic. Ideally it will be the real configuration as code